projects/igniteui-angular/src/lib/chips/themes/_base.scss

```diff
@@ -0,0 +1,412 @@
+
+        @include e(content) {
+            @include ellipsis();
+
+            max-width: 32ch;
+
+            &:empty {
+                display: none;
+            }
+        }
+
+        @include e(select) {
+            display: inline-flex;
+            align-items: center;
+            max-width: rem(24px);
+            opacity: 1;
+            z-index: 1;
+            transition: max-width .12s $ease-in-out-quad, opacity .12s $ease-in-out-quad .06s;
+            transition-behavior: allow-discrete;
+
+            @starting-style {
+                max-width: 0;
+                opacity: 0;
+            }
+        }
+
+        @include e(remove) {
+            display: inline-flex;
+            color: var-get($_theme, 'remove-icon-color', currentColor);
+
+            &:empty {
+                display: none;
+            }
+
+            &:focus {
+                igx-icon {
+                    color: var-get($_theme, 'remove-icon-color-focus');
+                }
+            }
+        }
+
+        @include e(ghost) {
+            position: absolute;
+            box-shadow: var-get($_theme, 'ghost-elevation');
+            overflow: hidden;
+            color: var-get($_theme, 'focus-text-color');
+            background: var-get($_theme, 'ghost-background');
+            // If z-index is not set,
+            //the chip would hide behind it's parent grid as it has a z-index of 1
+            z-index: 10;
+
+            igx-avatar,
+            igx-circular-bar,
+            igx-icon {
+                --component-size: var(--chip-size);
+            }
+
+            &:hover,
+            &:focus {
+                background: var-get($_theme, 'ghost-background');
+            }
+
+            &:focus {
+                .igx-chip__item {
+                    outline-style: none;
+                    color: var-get($_theme, 'focus-text-color');
+                    background: var-get($_theme, 'focus-background');
+                    border-color: var-get($_theme, 'focus-border-color');
+                }
+
+                .igx-chip__item--selected {
+                    color: var-get($_theme, 'focus-selected-text-color');
+                    background: var-get($_theme, 'focus-selected-background');
+                    border-color: var-get($_theme, 'focus-selected-border-color');
+                }
+            }
+
+            &:hover {
+                .igx-chip__item {
+                    color: var-get($_theme, 'hover-text-color');
+                    background: var-get($_theme, 'hover-background');
+                    border-color: var-get($_theme, 'hover-border-color');
+                }
+
+                .igx-chip__item--selected {
+                    color: var-get($_theme, 'hover-selected-text-color');
+                    background: var-get($_theme, 'hover-selected-background');
+                    border-color: var-get($_theme, 'hover-selected-border-color');
+                }
+            }
+        }
```

Comment:
Honestly, at this point, just to avoid having to manually refer to all these classes when the item also has a `.igx-chip__ghost` class, I'd simply make the ghost class a modifier class to t he original `.igx-chip__item` class, i.e. `.igx-chip__ghost` -> `.igx-chip__item--ghost`; In this way, we'd be able to do something like this:

```scss
@include e(item, $mods: (ghost, selected)) {
  /* styles when the item is both selected and a ghost */
}
```